resources/deploy-ghpages.sh

```diff
@@ -23,7 +23,7 @@ echo $HEADREV
 
 cd gh-pages
 git config user.name "Travis CI"
-git config user.email "github@fb.com"
+git config user.email "immutable-js-oss@gmail.com"
```

Comment:
Is there a better address to use here? Maybe `travis@travis-ci.com` or something? `@gmail` looks to me like a real address which a human would check, which this is not.

Reply:
There's one more place where that address was too, in case you update it.